frontend/src/app/docs/api-docs/api-docs-data.ts

```diff
@@ -9871,7 +9871,292 @@ export const restApiDocsData = [
         codeSampleBisq: emptyCodeSample,
       }
     }
+  },
+  {
+    type: "category",
+    category: "accelerator",
+    fragment: "accelerator",
+    title: "Accelerator",
+    showConditions: [""],
+    options: { officialOnly: true },
+  },
+  {
+    options: { officialOnly: true },
+    type: "endpoint",
+    category: "accelerator",
+    httpRequestMethod: "GET",
+    fragment: "get-deposit-history",
+    title: "GET Deposit History",
+    description: {
+      default: "<p>Returns a list of previous deposits made as prepayment for the accelerator service.</p>"
+    },
+    urlString: "/v1/services/accelerator/deposit-history",
+    showConditions: bitcoinNetworks,
+    showJsExamples: showJsExamplesDefaultFalse,
+    codeExample: {
+      default: {
+        codeTemplate: {
+          curl: `/api/v1/services/accelerator/deposit-history`,
+          commonJS: ``,
+          esModule: ``
+        },
+        codeSampleMainnet: {
+          esModule: [],
+          commonJS: [],
+          curl: [],
+          headers: "api_key: stacksats",
+          response: `[
+  {
+    "type": "Bitcoin",
+    "invoiceId": "CCunucVyNw7jUiUz64mmHz",
+    "amount": 10311031,
+    "status": "pending",
+    "date": 1706372653000,
+    "link": "/payment/bitcoin/CCunucVyNw7jUiUz64mmHz"
+  },
+  {
+    "type": "Bitcoin",
+    "invoiceId": "SG1U27R9PdWi3gH3jB9tm9",
+    "amount": 21000000,
+    "status": "paid",
+    "date": 1706372582000,
+    "link": null
+  },
+  ...
+]`,
+        },
+      }
+    }
+  },
+  {
+    options: { officialOnly: true },
+    type: "endpoint",
+    category: "accelerator",
+    httpRequestMethod: "GET",
+    fragment: "balance",
+    title: "GET Available Balance",
+    description: {
+      default: "<p>Return your current available balance, as well as how much funds are currently locked and much you've been charged so far.</p>"
+    },
+    urlString: "/v1/services/accelerator/balance",
+    showConditions: bitcoinNetworks,
+    showJsExamples: showJsExamplesDefaultFalse,
+    codeExample: {
+      default: {
+        codeTemplate: {
+          curl: `/api/v1/services/accelerator/balance`,
+          commonJS: ``,
+          esModule: ``
+        },
+        codeSampleMainnet: {
+          esModule: [],
+          commonJS: [],
+          curl: [],
+          headers: "api_key: stacksats",
+          response: `{
+  "balance": 99900000,
+  "hold": 0,
+  "feesPaid": 200000
+}`,
+        },
+      }
+    }
+  },
+  {
+    options: { officialOnly: true },
+    type: "endpoint",
+    category: "accelerator",
+    httpRequestMethod: "POST",
+    fragment: "estimate",
+    title: "POST Calculate Estimated Costs",
+    description: {
+      default: "<p>Return an estimation of the costs to accelerate a transaction.</p>"
```

Comment:
```suggestion
      default: "<p>Returns estimated costs to accelerate a transaction.</p>"
```

It would be good to explain what the different components of the pricing are...but I guess we'll have a FAQ for this at some point we can just link to from here.